src/components/TodoComponents/TodoForm.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Todo from './Todo';
+
+class TodoForm extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            item: ''
+        };
+    }
+
+    handleChanges = e => {
+        this.setState({
+            [e.target.name]: e.target.value
+        })
+    }
+
+    addItem = e => {
+        e.preventDefault();
+        this.props.addItem(this.state.item);
+        this.setState({
+            item: ''
+        })
+    }
+
+    render() {
+        return(
+        <div>
```

Comment:
The only recommendation that I have is to change this to a form and set you addItem function to onSubmit for the form. That way the user can press the button OR press enter to push the item to the list. It's just a better overall user experience.